Fish_Doku/Filter_Types/debug.md

```diff
@@ -2,36 +2,36 @@
 These filters print out the received values on the command line. Later they're supposed to
 post them to the debug table of the GUI.
 ## 8 bit output
-- Type name: `dbg:8bit`
+- Type name: `8bit`
 - Type id: `4`
-- Input channels: `value` of type `8bit`
+- Input channels: `value` of type `8bit` (default 0)
 - Output channels: None
 - Configuration parameters: None (Later: method to be used for gui communication)
 - initial parameters: None
 - GUI update keys: None
 
 ## 16 bit output
-- Type name: `dbg:16bit`
+- Type name: `16bit`
 - Type id: `5`
-- Input channels: `value` of type `16bit`
+- Input channels: `value` of type `16bit` (default 0)
 - Output channels: None
 - Configuration parameters: None (Later: method to be used for gui communication)
 - initial parameters: None
 - GUI update keys: None
 
 ## float output
-- Type name: `dbg:float`
+- Type name: `float`
 - Type id: `6`
-- Input channels: `value` of type `double`
+- Input channels: `value` of type `double` (default 0)
 - Output channels: None
 - Configuration parameters: None (Later: method to be used for gui communication)
 - initial parameters: None
 - GUI update keys: None
 
 ## color output
-- Type name: `dbg:pixel`
+- Type name: `pixel`
 - Type id: `7`
-- Input channels: `value` of type `hsv_pixel`
+- Input channels: `value` of type `hsv_pixel` (default white)
```

Comment:
We should also include (just for convenience purposes) the notation for white as a tuple of float values.